src/pretix/base/payment.py

```diff
@@ -78,6 +78,16 @@
 logger = logging.getLogger(__name__)
 
 
+class WalletQueries:
```

Comment:
This looks almost like an enum or almost like a models.TextChoices but isn't – is that on purpose?

Reply:
That was freestyled after PAYMENT_STATES... We can change this to make it look different or even drop it completely: It was just meant as a fixture to make it easier for plugins to use the correct string when requesting wallets...

Reply:
I'd like a enum.StrEnum more, but that only appeard in py3.11, so we can also keep it like this for now